upstream-community-operators/kubemq/0.5.3/kubemq-operator.v0.5.3.clusterserviceversion.yaml

```diff
@@ -0,0 +1,582 @@
+    * **Message Policy** - Each message can be configured with expiration and delay timers. In addition, each message can specify a dead-letter queue for unprocessed messages attempts
+    * **Long Polling** - Consumers can wait until a message available in the queue to consume
+    * **Peak Messages** - Consumers can peek into a queue without removing them from the queue
+    * **Ack All Queue Messages** - Any client can mark all the messages in a queue as discarded and will not be available anymore to consume
+    * **Visibility timers** - Consumers can pull a message from the queue and set a timer which will cause the message not be visible to other consumers. This timer can be extended as needed.
+    * **Resend Messages** - Consumers can send back a message they pulled to a new queue or send a modified message to the same queue for further processing.
+
+    ### Pub/Sub
+
+    KubeMQ supports Publish-Subscribe (a.k.a Pub/Sub) messages patterns with the following core features:
+
+    * **Events** -  An asynchronous real-time Pub/Sub pattern.
+    * **Events Store** -An asynchronous Pub/Sub pattern with persistence.
+    * **Grouping** - Load balancing of events between subscribers
+
+    ### RPC
+
+    KubeMQ supports CQRS based RPC flows with the following core features:
+
+    * **Commands** -  A synchronous two ways Command pattern for CQRS types of system architecture.
+    * **Query** - A synchronous two ways Query pattern for CQRS types of system architecture.
+    * **Response** - An answer for a Query type RPC call
+    * **Timeout** - Timeout interval is set for each RPC call. Once no response is received within the Timeout interval, RPC call return an error
+    * **Grouping** - Load balancing of RPC calls between receivers
+    * **Caching** - RPC response can be cached for future requests without the need to process again by a receiver
+
+    ## Interfaces
+
+    * **gRPC** - High performance RPC and streaming framework that can run in any environment, Open source and Cloud Native.
+    * **Rest** - Restful Api with WebSocket support for bi-directional streaming.
+
+    ## SDK
+
+    * **C#** - C# SDK based on gRPC
+    * **Java** - Java SDK based on gRPC
+    * **Go** - Go SDK based on gRPC
+    * **Python** - Python SDK based on gRPC
+    * **cURL** - cURL SDK based on Rest
+    * **Node** - Node SDK based on gRPC and Rest
+    * **PHP** - PHP SDK based on Rest
+    * **Ruby** - Ruby SDK based on Rest
+    * **jQuery** jQuery SDK based Rest
+
+    ## Getting Started
+
+    [Start with OpenShift](https://docs.kubemq.io/getting-started/create-cluster/openshift)
+
+    ## Get Enterprise Trial License
+
+    [Contact us](https://kubemq.io/contact-sales/)
+
+    ## Request a Demo
+
+    [See KubeMQ in Action](https://kubemq.io/request-a-demo/)
+  version: 0.5.3
+  replaces: kubemq-operator.v0.5.2
```

Comment:
@kubemq Can you fix it to `kubemq-operator.v0.5.1`? Since only this version is the last in repository.